styles.scss

```diff
@@ -0,0 +1,251 @@
+$backgroundColor: #f5f5f5;
```

Comment:
Nebála bych se oddělit (komentářem, odřádkováním apod.), kde začínají a končí barvy, velikosti fontů a proměnné s jiným účelem. Čistě kvůli čitelnosti. :) Ve velkých projektech by se to dělalo i samostatnými soubory (jeden soubor na barvy a tak dále).